bool limit_az
bool limit_el
bool emergency
float64 command_az
float64 command_el

